src/views/nads/details/tabs/yaml/NADYAMLPage.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+
+import { ResourceYAMLEditor } from '@openshift-console/dynamic-plugin-sdk';
+import { Bullseye } from '@patternfly/react-core';
+import { NetworkAttachmentDefinitionKind } from '@utils/resources/nads/types';
+import Loading from '@utils/components/Loading/Loading';
+
+type NADYAMLPageProps = {
+  obj?: NetworkAttachmentDefinitionKind;
+};
+
+const NADYAMLPage: FC<NADYAMLPageProps> = ({ obj: nad }) => {
+  const loading = (
+    <Bullseye>
+      <Loading />
+    </Bullseye>
+  );
+  return !nad ? (
```

Comment:
I prefer to see an extra line above `return` block, to make the code even more readable, but it's nothing so important. Up to you :)